addons/mail/tests/test_rtc.py

```diff
@@ -451,6 +451,7 @@ def test_30_add_members_while_in_call_should_invite_new_members_to_call(self):
                 (self.cr.dbname, 'res.partner', self.user_employee.partner_id.id),  # update of last interest (not asserted below)
                 (self.cr.dbname, 'res.partner', test_user.partner_id.id),  # update of last interest (not asserted below)
                 (self.cr.dbname, 'mail.channel', channel.id),  # new member (guest) (not asserted below)
+                (self.cr.dbname, 'mail.guest', test_guest.id),
```

Comment:
comment it similar to the others so we can know what is going on `# channel joined for guest (not asserted below)`